js/demo.js

```diff
@@ -0,0 +1,17 @@
+import $ from 'jquery';
```

Comment:
I'm not sure what all this is for? Why do we need to load the header and footer with JS?

If it must, which I'm not convinced it does, we don't need jQuery or Mustache. 

Remove jQuery in favour or native/polyfilled Promise and fetch:

```
Promise.all([
   fetch('/partials/header.html'),
   fetch('/partials/footer.html')
]).then([header, footer] => {
  document.getElementById('ig-header').innerHTML = header;
  document.getElementById('ig-footer').innerHTML = footer;
  igArticle.init();
})